source/03-components/wysiwyg/wysiwyg.es6.js

```diff
@@ -0,0 +1,31 @@
+import Drupal from 'drupal';
+import once from 'once';
+
+Drupal.behaviors.wysiwyg = {
+  attach(context) {
+    once('wysiwyg', '[data-wysiwyg]', context).forEach(wysiwyg => {
+
+      // Wrap all tables output via WYSIWYG with a responsive table layout.
+      const tables = wysiwyg.querySelectorAll('[data-wysiwyg] table');
+      tables.forEach((table, i) => {
+        const responsiveTable = document.createElement('div');
+        const caption = table.querySelector('caption');
+
+        responsiveTable.classList.add('l-responsive-table');
+        responsiveTable.setAttribute('tabindex', '0');
+        responsiveTable.setAttribute('role', 'region');
```

Comment:
@dcmouyard as I was working on this to port it to GUSWDS, I realized it's possible that someone could have already manually added the responsive layout div around their table and this would add a second one. Do you think it needs to a check to make sure it's direct parent isn't already a responsive table wrapper div?

Reply:
@coreylafferty Good catch! That edge case should be handled properly now.